pan3d/dataset_builder.py

```diff
@@ -1,10 +1,10 @@
 import os
 import json
-import importlib
 import pyvista
 import xarray
 
 from pan3d.utils import coordinate_auto_selection
+from pan3d.catalogs import call_catalog_function
```

Comment:
ideally you could just do `from pan3d import catalogs` and then use it down the road like `catalogs.get("esg")` or `catalogs.load("esg", id)`

I made up the name of the methods but they should match what is the intent. Also since you grab them from `catalogs` you don't need to repeat catalog in the method name...

Reply:
I made changes in [9bb4cfd](https://github.com/Kitware/pan3d/pull/69/commits/9bb4cfdb065ac676a7ac12cb9777706161b56794). Is this what you mean?

Reply:
yes exactly. Now the code is easier to read and follow